active/radiobuttons/RadioButtonsAPI.md

```diff
@@ -0,0 +1,155 @@
+# Background
+
+RadioButtons is our answer to enabling better keyboarding and narrator support for RadioButtons that need to be presented in a related list.
+
+Currently, the only way to group RadioButtons is through using a StackPanel and individually labeling each RadioButton through its GroupName property in order for narrator to read out the options somewhat understandably.
+
+However, using that method still left gaps in the narrator experience and StackPanel does not behave the way a user would expect when navigating the list via keyboard - RadioButtons was introduced to help solve those issues.
+
+# Description
+
+RadioButtons is a new control that enables you to create related lists of RadioButtons easily, while also fully supporting keyboarding and narrator functionality.
```

Comment:
I think it is confusing that RadioButtons is a control, but we are referring to collections of objects of the RadioButton class as RadioButtons in this spec as well. Perhaps we could be a little more verbose to avoid this confusion by replacing the RadioButtons which do not refer to the new control with something like "RadioButton Elements".

Reply:
I see two comments of your that look very similar - are you suggesting either using "RadioButton Elements" or "Groups of RadioButton objects"? Either way I think I understand what you're getting at and will try and re-word it to be less confusing.

Reply:
Sorry, I thought I deleted the other one. Personally I think RadioButton Elements is cleaner.  The main point is that because of the name of this control I don't think we should use RadioButtons as the plural of RadioButton, since we are trying to name this new control RadioButtons.

Reply:
Makes sense! I just went through and updated the wording in those sections - take a look and let me know if that helps make it clearer.

Reply:
Just one more, made a separate comment with the suggested change.

Reply:
Noted! I committed them.